pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -113,6 +113,10 @@ var (
 	zone1Labels = map[string]string{v1.LabelZoneFailureDomain: "us-east-1", v1.LabelZoneRegion: "us-east-1a"}
 )
 
+func init() {
+	klog.InitFlags(nil)
```

Comment:
I don't see this used in other unit tests. Is this necessary?

Reply:
Without it, one cannot enable log output when running the test manually. The default log level is unchanged, so nothing changes for the Prow CI, but when running manually one can now do:
```
go test -v ./pkg/controller/volume/scheduling -args -v 5
```

Note the `-v 5`.

When the test was written, glog flags where always enabled. That changed when moving to klog.

Reply:
This is fine for this particular test, but we should probably make the change in all tests? I'm not sure if there's an easier way to update every test.

Reply:
Yes, it may be worthwhile to do it in all tests at once instead of on demand. Some other developers might not realize that they can get additional logging when trying to debug something.

I don't think there's a way to do this in just one file. We could limit it to a one-line change in each test with a helper package that contains the init function, but that may be overkill:

`test/util/klog/klog.go:`
```
import k8s.io/klog
package klog

func init () {
    klog.InitFlags(nil)
}
```

Then in a test, import that package instead of `k8s.io/klog`. Do you prefer that or explicit `init` functions?

Reply:
cc @kubernetes/k8s-infra-team for suggestions

Reply:
I didn't even know this was an alias but apparently it CCs me :upside_down_face: 
This is probably a question for #testing-commons on slack.

Reply:
The comment on #testing-commons was that separate init functions are preferred over import magic.